packages/gatsby/src/utils/__tests__/get-latest-apis.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
```

Comment:
I want to use `require` in order mock objects to be inferred as any. I came up with using `as jest.Mock<>` as an alternative but I guess this is a bit redundant and no different from using `any`.